test/testcafe/spec/DeviceChallengePollView_spec.js

```diff
@@ -316,9 +316,14 @@ async function setupLoopbackFallback(t, widgetOptions) {
   return deviceChallengeFalllbackPage;
 }
 
+const skipTest = () => {
+  return null;
+  // return test.skip;
+};
```

Comment:
Explain reasoning

Reply:
I used DevTools to inspect network requests and saw that test `in loopback server approach, will cancel polling when challenge errors out` mistakenly uses request hooks of previous (skipped) test (`loopbackPollTimeoutMock`)
<img width="949" alt="Screenshot 2024-06-18 at 19 01 37" src="https://github.com/okta/okta-signin-widget/assets/72614880/083de3e8-8a34-4c35-b11e-90629d9d54ed">
Request to `:2000/probe` has status 200, but should have 500.
This issue can be solved by just removing / commenting out skipped tests